utils/trackerbot.py

```diff
@@ -146,7 +146,7 @@ def mark_provider_template(api, provider, template, tested=None, usable=None,
     Returns the response of the API request
 
     """
-    provider_template = _as_providertemplate(provider, template)
+    provider_template = _as_providertemplate(provider, template, group=stream)
```

Comment:
I think this portion, connecting the stream from `trackerbot_add_provider_template` to `mark_provider_template`,  will need more testing on the various streams.

While testing the upstream_stable changes, the templates were getting added to the unknown group, and staying there after sync_template_tracker.py was run because the template already existed and `parse_template` was never called.

This connected the stream provided to the group, and the template was marked correctly in trackerbot.

@jkrocil I could use your help determining what's happening with trackerbot, and the templates not getting identified for the right group (upstream_stable in this case).